src/index.tsx

```diff
@@ -19,14 +19,25 @@ function findGetParameter(parameterName: string): string {
     return result;
 }
 
-console.log(findGetParameter("test"));
+const dbId = findGetParameter("dbid")
+
+if (!dbId) {
+    window.alert('Please add the "dbid" query paramenter to the URL.')
+    throw new Error('dbid query paramenter not set in URL')
```

Comment:
this is not great, ideally we offer an input form for the configuration (if set: allow to edit, unset: don’t render page and apolloClient until set …) – but can keep this as FLUP